activitysim/defaults/tables/skims.py

```diff
@@ -39,3 +39,17 @@ def sovmd_skim(nonmotskm_matrix):
 def sovpm_skim(nonmotskm_matrix):
     # FIXME use the right omx file
     return skim.Skim(nonmotskm_matrix, offset=-1)
+
+
+@sim.injectable(cache=True)
+def skims():
+    skims = skim.Skims()
+    # FIXME - this is reusing the same skim as all the different kinds of skims
+    for typ in ["SOV_TIME", "SOVTOLL_TIME", "HOV2_TIME",
```

Comment:
In this specific case, I'm really just reading in the distance skim 15 times so that I have skims to play with ;)  But in a more general situation, I think it's interesting to know what is "core" and what is "configuration."  I'm not certain there's a bright red line.  I mean, in many cases I think it's easier to  "configure" stuff directly in Python rather than having to go to a second location to find out what something is - most functions are now at most 15 lines long and having all the code in one place is pretty nice.  On the other hand if something is used more than once, it should probably be defined once and shared, whether it's defined in Python or a text file.  

So far I've set a principle for myself that models should be very generic, should be shared among regions, should only be written by a certain class of users/programmers, etc.  But tables (data sources) and columns mostly just use Pandas and I think should be fair game for a much larger class of users.  As long as a variable is defined only once (which the simulation framework is designed to enable exactly that), I think it's fair game to "configure" a variable in Python.

But this principle is still sort of a work in progress and is perhaps not correct.  Or perhaps it's just not clear that's what is intended.  I think it would require moving the stuff in the "models" directory into the "activitysim" directory and keeping tables/column under defaults.  It is interesting to me to know how much (if any?) of the the variable definitions are shared among regions.  I'm guessing not much.  But I'm sure models can run the same way and can be shared between regions with no edits in the code.

Again, this was true of CTRAMP in a major way as well.  There are constant references to methods in the code for things that can't be computed in the CSV files.  In our version it's just that those Python definitions are easier to write, are shorter, and don't require compilation, so are a lot more accessible to the modelers.  I'm trying to keep all of these things under the "tables" directory, but a few methods should maybe be moved up the stack or down for sure.